stages/org.osbuild.chrony.meta.json

```diff
@@ -19,28 +19,19 @@
     "  - 'maxpoll'",
     "  - 'iburst' (defaults to true)",
     "  - 'prefer' (defaults to false)",
+    "",
     "The `leapsectz` option configures chrony behavior related to automatic checking",
     "of the next occurrence of the leap second, using the provided timezone. Its",
     "value is a string representing a timezone from the system tz database (e.g.",
     "'right/UTC'). If an empty string is provided, then all occurrences of",
     "`leapsectz` directive are removed from the configuration.",
-    "Constraints:",
-    "  - Exactly one of 'timeservers' or 'servers' options must be provided."
+    "",
+    "The refclock directive can be used to specify one or more hardware",
+    "reference clocks to be used as a time source."
   ],
   "schema": {
     "additionalProperties": false,
-    "oneOf": [
-      {
-        "required": [
-          "timeservers"
-        ]
-      },
-      {
-        "required": [
-          "servers"
-        ]
-      }
-    ],
```

Comment:
The original idea was that one can't specify both options, which would still make sense. Not a blocker from my side. It is just that `servers` is a simplified version of `timeservers` that was added originally when this stage was introduced.

Reply:
I don't mind making it less strict. These two will result in the same config directive being used in the Chrony config, but `servers` is a limited version, and it was kept there mostly for backward compatibility.

Reply:
Maybe we should start versioning stages so we can drop options :)